nexus/db-queries/src/db/datastore/volume.rs

```diff
@@ -2160,6 +2469,71 @@ fn replace_region_in_vcr(
     Ok(new_vcr)
 }
 
+/// Replace a read-only target in a VolumeConstructionRequest
+///
+/// Note that UUIDs are not randomized by this step: Crucible will reject a
+/// `target_replace` call if the replacement VolumeConstructionRequest does not
+/// exactly match the original, except for a single Region difference.
+///
+/// Note that the generation number _is not_ bumped in this step.
+fn replace_read_only_target_in_vcr(
+    vcr: &VolumeConstructionRequest,
+    old_target: SocketAddrV6,
```

Comment:
Same as above, possibly useful to have newtype wrappers for safer calling. If you can make them not implement `From` that would prevent bugs by calling `.into()` without being explicit at the callsite.

Reply:
done in 7c71482c5, using the same newtype :)

Reply:
Awesome.